### PR TITLE
[#417] Use Google Maps API for location autocomplete

### PR DESCRIPTION
**Pull request by julianguyen**
https://github.com/julianguyen/ifme/issues/417

- Fixes blocked HTTP request for location autocomplete that exists with GeoBytes
- Remove timezone from user account form

## Sign in form
image

## Accounts form
image

### CODE REVIEW

File: app/assets/javascripts/get_location.js.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function initAutocomplete() {
```
Comment (julianguyen) [inline app/assets/javascripts/get_location.js.erb]: Had to convert this to a .js.erb file so that I could pull out the Rails environment variable!

Comment (nma) [inline app/assets/javascripts/get_location.js.erb]: neat!

File: spec/javascripts/GetLocationSpec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+describe("GetLocation", function() {
+var src = 'https://maps.googleapis.com/maps/api/js?libraries=places&callback=initAutocomplete';
+
+describe("onReadyGetLocation", function() {
+  it("does not Google Maps autocomplete script to page", function() {
```
Comment (nma) [inline spec/javascripts/GetLocationSpec.js]: does not load?

File: README.md [hunk 3]
```diff
@@ -236,7 +236,9 @@ The following are not mandatory, but are required if you would like to test/use
...
+[Google OAuth 2.0 IDs](https://console.developers.google.com) is used for OAuth (Sign in with Google) and Calendars (refill dates for Medications). If you would like to use this feature in your local environment, please create your own account, generate keys, and update `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` in `config/env/test.env` and `config/env/development.env`. You'll need activate both the Google+ API and the Contacts API for OAuth, and the Calendar API for Calendars. Under the credentials tab, make sure to add the Authorized redirect URI as `http://localhost:3000/users/auth/google_oauth2/callback`. Note, you may have to hit the Save button twice for this to take effect.
+
+[Google API](https://console.developers.google.com) is used for location autocomplete. If you would like to use this feature in your local environment, please create your own account, generate keys, and update `GOOGLE_API_KEY` in `config/env/test.env` and `config/env/development.env`.
```
Comment (nma) [inline README.md]: will need a comment about activating `Google Maps JavaScript API` as I had issues running this branch without.

**Comment by julianguyen** [comment]
Thanks for reviewing!!! :D

**Pull request merged**